### ISSUE TITLE
"Last Edit" granularity is misleading

### ISSUE THREAD

**Issue by mojodna** [developmentseed/scoreboard#29]
`getLastEdit` in https://github.com/developmentseed/scoreboard/blob/bdd5822cf28b6e6ca9546a9246ba2db04eec8dff/frontend/src/components/UserHeader.js#L15-L19 uses the list of `edit_times` to surface the most recent edit.

`edit_times` in the API response (`api/users/<id>`) only includes dates (no timestamps), so this will be misleading if the last edit is in the window for which `distanceInWordsToNow` renders dates in granularity smaller than days:

```javascript
{
  // ...
  "records": {
    edit_times: [
      // ...
      {
        "day": "2018-09-21",
        "count": 2
      }
    ]
    // ...
  }
}
```

Compare to https://github.com/developmentseed/scoreboard/blob/bdd5822cf28b6e6ca9546a9246ba2db04eec8dff/frontend/src/components/AllUsersTable.js#L30, which uses the `api/users` endpoint and has access to a `last_edit` timestamp represented as epoch milliseconds.

**Comment by mojodna** [comment]
`last_edit` in `api/users` may appear to have greater granularity, but is aligned to the beginning of a UTC day.

Digging in a bit more, the limitation leading to the above ambiguity originates in the OSMesa API, as no single `last_edit` is returned (or is available in the materialized view as of now), just `edit_times`.

/cc @moradology

**Comment by rossbernet** [comment]
![image](https://user-images.githubusercontent.com/8103418/45980533-3413cf00-c020-11e8-93c9-da7bf74aff0d.png)

![image](https://user-images.githubusercontent.com/8103418/45980544-3b3add00-c020-11e8-9f81-78a2d526f310.png)

I believe this issue is a result of a comment I made about the last edits. 

I had just signed up for an account and made my first edit. Then it said "about 20 hours" in one place, and on the next page it said 16 hours.
